AstSemantics.md

```diff
@@ -248,6 +299,32 @@ The details of index space for local variables and their types will be further c
 e.g. whether locals with type `i32` and `i64` must be contiguous and separate from
 others, etc.
 
+## Global variables
+
+A *global variable* stores a single value of a fixed [value type](#types) and may be
+declared either *mutable* or *immutable*. This provides WebAssembly with memory
+locations that are disjoint from any [linear memory](#linear-memory) and thus
+cannot be arbitrarily aliased as bits.
```

Comment:
Is the intent that when we support multiple memories those can also be loaded as immutable? That seems very useful for a bunch of usecases, and orthogonal to `mprotect`.